pkg/policy/validate.go

```diff
@@ -1259,6 +1259,11 @@ func validateExcludeResourceDescription(rd kyverno.ResourceDescription) (string,
 // field type is checked through openapi
 func validateResourceDescription(rd kyverno.ResourceDescription) error {
 	if rd.Selector != nil {
+		for _, v := range rd.Selector.MatchLabels {
+			if v == "*" || strings.HasPrefix(v, "*") || strings.HasSuffix(v, "*") {
```

Comment:
I think we need to clear what can be supported here, `*` can exist anywhere in a string:
- "*"
- "*pattern"
- "pattern*"
- "pat*tern"

What do we want to support?

Reply:
Currently we are supporting, 1st three patten mentioned. we should support  **pat*tern**?

Reply:
I'm not sure, what's the requirement?

Reply:
we can support this patten to

-  "*pattern*"

Reply:
Please list all the supported patterns in the PR description and create the issue to track dock updates.